http-server-spec/Rakefile

```diff
@@ -21,6 +21,15 @@ task :test do
 end
 
 namespace :test do
+  desc "Run all of the tests for implemented features"
+  task :implemented do
```

Comment:
Yes -- Kseniia showed me how to add this task and tag selected tests so that when I run the CI, it will only run tests for features that have been implemented

Reply:
Is there a specific command to run other than `rake test`? I'm seeing all of the tests being ran, but I also might be missing something

Reply:
it runs with the command `rake test:implemented`

Reply:
`rake test` will still run the entire acceptance test suite but this allows me to run `rake test:implemented` in my CI workflow so that it doesn't fail for features that haven't been implemented yet